src/app/shared/utils/twinAndJsonSchemaDataConverter.ts

```diff
@@ -107,7 +107,7 @@ export const findPathsTowardsMapType = (
 };
 
 export const getNumberOfMapsInSchema = (settingSchema: ParsedJsonSchema): number => {
-    const hasMatch = JSON.stringify(settingSchema).match(/additionalProperties/g);
+    const hasMatch = settingSchema && JSON.stringify(settingSchema).match(/additionalProperties/g);
```

Comment:
notice a number of these checks have been removed.  Can you help me understand why they are no longer required?

Reply:
Can you point me to where these has been removed? I am adding the null check here, not trying to remove anything.